libcore/gof-directory-async.vala

```diff
@@ -1030,8 +1033,8 @@ public class Async : Object {
             GLib.File from = pair.index (0);
             GLib.File to = pair.index (1);
 
-            list_from.append (from);
-            list_to.append (to);
+            list_from.prepend (from);
+            list_to.prepend (to);
         }
```

Comment:
The order of these lists is not important; the files listed are removed to and added from structures (HashTable, ListModel) where the sequence of changes have no effect.